__tests__/app.test.js

```diff
@@ -111,6 +111,28 @@ describe("GET /api/articles", () => {
         });
       });
   });
+
+  test("200: filter articles by existing topic", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toBe(true);
+        body.forEach((article) => {
+          expect(article.topic).toBe("cats");
+        });
+      });
+  });
+
+  test("200: filter articles by non existing topic, returns all articles", () => {
+    return request(app)
+      .get("/api/articles?topic=radiohead")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toBe(true);
+        expect(body).toBeSortedBy("created_at", { descending: true });
+      });
+  });
```

Comment:
This should be a 404 error because it's not found - I can see what you're trying to do though. In front end you could still use the error to revert the page back to all articles. But my advice would be for now that the topic does not exist so we should get an error. 

One more test you can add - there's a topic that exists but has no articles attached to it yet. You should get a 200 and empty array!